packages/gatsby/src/joi-schemas/joi.js

```diff
@@ -1,10 +1,18 @@
 const Joi = require(`joi`)
 
+const stripTrailingSlash = chain => chain.replace(/(\w)\//, `$1`)
```

Comment:
This is a nice addition - rather than requiring documentation (e.g. don't add a trailing slash), this will just remove it for us.

It only impacts a string with content, e.g. `/` is left alone.